oci-unit-tests/telegraf_test.sh

```diff
@@ -104,4 +104,11 @@ test_manifest_exists() {
     assertTrue "Manifest file(s) do(es) not exist in image" $?
 }
 
+test_all_supported_architectures_are_available() {
+    debug "Testing that the supported architectures are available in the manifest"
+
+    check_supported_architectures
+    assertTrue "Some supported architectures are not present in the registry" $?
```

Comment:
Any chance we could list the missing arches in this test output and/or list the manifest list? This would likely aid debugging in the future.

Reply:
Yes, good point.